src/runtime-rs/crates/resource/src/cgroups/mod.rs

```diff
@@ -33,22 +33,49 @@ pub struct CgroupConfig {
     pub path: String,
     pub overhead_path: String,
     pub sandbox_cgroup_only: bool,
+    pub is_threaded_mode: bool,
+    pub specified_controllers: Option<Vec<String>>,
 }
 
 impl CgroupConfig {
     fn new(sid: &str, toml_config: &TomlConfig) -> Result<Self> {
-        let overhead_path = utils::gen_overhead_path(sid);
+        let mut overhead_path = utils::gen_overhead_path(sid);
         let spec = load_oci_spec()?;
-        let path = spec
+        let mut path = spec
             .linux
             // The trim of '/' is important, because cgroup_path is a relative path.
             .map(|linux| linux.cgroups_path.trim_start_matches('/').to_string())
             .unwrap_or_default();
 
+        path = utils::rename_cgroup_path(&path);
+        overhead_path = utils::rename_cgroup_path(&overhead_path);
+
+        // In cgroup v2, the sandbox and overhead cgroup in threaded mode are placed under
+        // the same cgroup in domain threaded mode.
+        // In this way, vCPU threads and VMM processes can be separated into two cgroups.
+        // For details, please refer to host cgroups design document
+        // https://github.com/kata-containers/kata-containers/blob/main/docs/design/host-cgroups.md.
```

Comment:
Do not use main/master in GitHub URL, use tag/commit instead (?

Reply:
Thanks for your review. @YushuoEdge

Is it to prevent the link from being incorrectly accessed due to changes in files on the main branch?

After https://github.com/kata-containers/kata-containers/pull/5153 is merged, I will write the file link of the docs commit in the comments here.

Reply:
Yes, thanks!